distributions/nuxt-ts/tsconfig.json

```
@@ -0,0 +1,33 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "moduleResolution": "node",
+    "lib": [
+      "esnext",
+      "esnext.asynciterable",
+      "dom"
+    ],
+    "esModuleInterop": true,
```

@kevinmarrec might be worth to add `resolveJsonModule": true,` to allow frictionless import of json files
```suggestion
    "esModuleInterop": true,
    "resolveJsonModule": true,
```